robotino_controller_configuration_gazebo/package.xml

```diff
@@ -13,6 +13,8 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostopic</build_depend>
   <build_depend>gazebo_ros_control</build_depend>
+  <build_depend>ros-indigo-gazebo-plugins </build_depend>
```

Comment:
does it work? I though that  to the package.xml you can only add the ros name of the packages and not the ubuntu distro name. 

and where are used on squirrel_controller_configuration_gazebo the gazebo_plugins package?